src/components/FileUploader/styles.js

```diff
@@ -0,0 +1,86 @@
+import styled, { css } from 'styled-components';
+import FormElementWrapper from '../FormComponents/FormElementWrapper';
+import UiText from '../UiText';
+
+const styles = {
+  textColor({ invalid, valid }) {
+    if (invalid) {
+      return 'var(--f-color-element--error)';
+    }
+    if (valid) {
+      return 'var(--f-color-element--success)';
+    }
+    return 'var(--f-color-text--secondary)';
+  },
+  borderColor({ invalid, valid, hasFile }) {
+    if (hasFile) {
+      return 'transparent';
+    }
+    if (invalid) {
+      return 'var(--f-color-element--error)';
+    }
+    if (valid) {
+      return 'var(--f-color-element--success)';
+    }
+    return 'var(--f-color-element--tertiary)';
+  },
+};
+export const StyledFileUploader = styled(FormElementWrapper)`
+  position: relative;
+  width: 100%;
+  flex-direction: column;
+  margin: var(--f-space--m) 0;
+  border-radius: var(--f-borderRadius--l);
+  color: ${styles.textColor};
+  background-color: var(--f-color-element--primary);
+  transition: all 0.15s ease-out;
+  border: 1px dashed ${styles.borderColor};
+  height: ${({ hasFile }) =>
+    hasFile ? 'auto' : 'var(--f-FileUploader-height)'};
+  text-align: ${({ hasFile }) => hasFile && 'center'};
+
+  ${({ hasFile }) =>
+    !hasFile &&
+    css`
```

Comment:
I don't think you have to wrap this into `css`?

Reply:
True, but without it we don't get syntax highlight 😅. I don't think it makes any difference once compiled

Reply:
https://github.com/heetch/flamingo/pull/135/commits/1168170e55ce7c172e152c368afd21c7c5ae84c4

You inspired me 😄

Reply:
😍